src/serial/serialcommands.cpp

```diff
@@ -248,9 +290,9 @@ namespace SerialCommands {
 			if (scanRes >= 0) {
 				logger.info("[WSCAN] Found %d networks:", scanRes);
 				for (int i = 0; i < scanRes; i++) {
-					logger.info("[WSCAN] %d:\t%02d\t%s\t(%d)\t%s",
+					logger.info("[WSCAN] %d: %02d '%s' (%d dBm) %s",
```

Comment:
the original reason for the usages of `\t` and no `'` or ` ` is because it simplified parsing as the only people that would be able to input tab in their SSID would be people that know what the hell they are doing, a `'` or a ` ` could easily be put on the SSID and break parsing

Reply:
Switched to tabs now, the `'` can be considered to always surround the SSID and be trimmed (and/or use the length printed before)